blockchains/smartchain/assets/Logo Information update

```diff
@@ -0,0 +1 @@
+
```

Comment:
{
    "name": "SHIBA RISE",
    "type": "BEP20",
    "symbol": "SHIBARISE",
    "decimals": 9,
    "website": "https://www.shibarise.xyz/",
    "description": "SHIBARISE is an experiemental community driven crypto-currency with buy, burn and reflect tokenomics that automatically distribute 2% to holders, 6% the whale wallet and 3% to marketing. SHIBARISE is a rose growing in the concrete. A trustworthy dev, healthy community, long term plans and more. It's not just a currency, its a community and vision. Plans to develop major game changers seek more info within whitepaper. ",
    "explorer": https://bscscan.com/token/0x0766d199d90b3c1ae34cb12281ec83142abcca0a
    "status": "active",
    "id": "0x0766d199D90b3C1Ae34CB12281Ec83142ABCCa0A"
}